docs/proposals/accepted/observatorium.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "2021-05: Observatorium"
+menu:
+  docs:
+    parent: "acc-proposals"
+toc: true
+---
+
+Taken from [Open Source Design Doc Template](https://docs.google.com/document/d/1zeElxolajNyGUB8J6aDXwxngHynh4iOuEzy3ylLc72U/edit#).
+
+## Observatorium
+
+Initial Observatorium Design Proposal
+
+* **Owners:**
+  * `@bwplotka`
+  
+* **Related Tickets:** 
+  * `<JIRA, GH Issues>`
+  
+* **Other docs:** 
+  * RH internal: [1](https://docs.google.com/document/d/1lX0Tl77NFp9m1ZhV3ya1iOQSLZUI0fYbliTt0H_WGJA/edit), [2](https://docs.google.com/document/d/15MZfUPXvOMc9rUlXOVmveQMgiWVzgzNsRwxR8tZmx4s/edit)
+
+## TL;DR
+
+Observatorium composes multiple separate backends for each signals that shares similar deployment and installation strategy into single coherent multi-signal backend with consistent APIs for metrics, logging, tracing.
+
+The idea is to have easy to operate stack that's easy to run as SaaS as well as for your own use. 
+
+## Why
+
+Undoubtedly, Observability is critical for developing and running software, especially given Cloud Native application’s complexities and scale. Traditional Observability signals like metrics, logging, tracing, and profiles are key to enable monitoring (with alerting), debugging, billing, incident handling, reporting, analytics, and more. All that is required to make data-driven decisions.
+
+Yet the biggest benefit from Observability is when we [deploy multiple signals together into one coherent and consistent system](https://www.bwplotka.dev/2021/correlations-exemplars/).
+
+### Pitfalls of the current solution
+
+All the currently available alternatives and reasons why such Observability design might be a better idea are stated in [Alternatives Section](#Alternatives).
+
+## Goals
+
+**Target Audience**: Engineers who want to provide observability services for monitoring, debugging and SLOs assurance for themselves, their teams or even SaaS customers.
+
+An Observability System that has the following characteristics:
+
+* Open Source first
+  * Apache v2 license
+  * Free to use
+  * All (in-scope) contributions welcome.
+* Ability to scrape/ingest (write), store, query (read), and retrieve metrics, logs and traces from multiple clusters and tenants. APIs for now:
+  * Metrics
+    * Write:
+      * OpenMetrics (pull model)
+      * Prometheus Remote Write API (push model)
+    * Read:
+      * Compatible with the Prometheus query APIs (e.g PromQL) making it possible to use any dashboard solution (e.g. Grafana).
+  * Logs: Kubernetes and Pod logs (Loki API)
+* Support recording rules and alerting based on data from multiple clusters and tenants.
+  * Allow users to configure alerts and recording rules with immediate effect (minutes)
+* Unified installation and deployment model for Kubernetes and OpenShift using Operator, OLM, non Operator, helm and Openshift Templates. 
+* Support multi-tenancy for all APIs allowing to choose between soft and hard tenancy.
```

Comment:
If Observatorium is an opinionated SaaS, should we not mandate one of {soft,hard} tenancy? (probably soft) That way we limit the number of options we need to consider when designing & implementing features.

Reply:
Yea, but our opinion is to use hard a lot, so we need both IMO (: